content/blog/2023-05-26-case-study-xiatech/index.md

```diff
@@ -0,0 +1,86 @@
+  - key: Employees
+    value: 70+
+  - key: AWS Services
+    value:
+      - Lambda
+      - SQS
+      - Kinesis
+      - DocumentDB
+      - DynamoDB
+  - key: Integrations
+    value:
+      - Testcontainers
+      - Terraform
+      - Terragrunt
+---
+<div class="quote-container mt-4">
+
+  > _“Our engineering team utilizes LocalStack to provide a complete, localized AWS environment where developers can build, test, profile and debug infrastructure and code ahead of deployment to the cloud.”_
+  <div class="quote-author">
+    <p><a href="https://www.linkedin.com/in/rick-timmis-6a1437/">Rick Timmis</a>,</p>
+    <p>Head of Engineering at <a href="https://www.xiatech.co.uk/">Xiatech</a></p>
+  </div>
+</div>
+
+<div class="lead-content">
+  <p>Xiatech helps organizations and users connect systems, unify data, and make data-driven decisions with artificial intelligence via a continuous flow of actionable insights. Xiatech’s flagship platform is Xfuze, a hyper-integration platform that helps customers shorten to insight, create value from data and turbocharge legacy technology investments. Xfuze integrates with various systems like customer relationship management (CRM), enterprise resource planning (ERP), logistics, warehousing, marketing, and different cloud-based systems. Xfuze creates a central, singular view of data which then helps provide automation, data management, continuous insights and advanced analytics — all in one solution.</p>
+
+  <p>Xiatech deploys Xfuze across cloud providers, including Amazon Web Service (AWS) and Google Cloud Platform. Their Hyper-Integration platform utilizes an event driven architecture, and leverages many distributed computing technologies, including SQL/NoSQL databases, cloud functions, and AWS Serverless technologies. Cloud infrastructure is orchestrated using Terraform, and Terragrunt to co-ordinate the compute resources upon which Xfuze stands.</p>
+
+  <p>We spoke with Rick Timmis, Head of Engineering at Xiatech, to learn more about how LocalStack has helped their organization accelerate its cloud development workflows with LocalStack and empowered engineers to be independent of their local development and test toolings.</p>
+</div>
+
+## Challenge
+
+The challenge for engineers includes vast amounts of technologies, ranging from SOAP or XML APIs to RESTful services, and the need to provide integrations with Lambda, SQS, Kinesis, DocumentDB, DynamoDB and more. As an engineer, orchestrating these serverless technologies via a command-line interface and developing/debugging the services was very slow and tedious. 
+
+To solve this problem, Xiatech created a developer-experience (DevEx) environment to bring forward a localized environment on the individual machines used by engineers to enable them to run integrations between one system and another. It would allow them to run these simulations and profile, debug, and check the memory state and the stack — all on their IDEs. 
```

Comment:
```suggestion
To solve this problem, Xiatech created a developer-experience (DevEx) environment to bring forward a localized environment on the individual machines used by engineers to enable them to run integrations between one system and another. It would allow them to run these simulations and profile, debug, and check the memory state and the call stack — all from within their IDEs. 
```